### PR TITLE
Rmt server entrypoint improvements

### PR DESCRIPTION
**Pull request by dcermak**

### CODE REVIEW

File: src/bci_build/package/rmt.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
         ],
         version_in_uid=False,
         support_level=SupportLevel.L3,
-        package_list=["rmt-server", "catatonit"],
+        package_list=["rmt-server", "catatonit", "bash"],
```
Comment (alexandrevicenzi) [inline src/bci_build/package/rmt.py:34]: Q: bash is not part of the base image already?

Comment (dcermak) [reply src/bci_build/package/rmt.py:34]: It is, but this is a safety hatch, in case anyone decides to switch to a micro based build

Comment (alexandrevicenzi) [reply src/bci_build/package/rmt.py:34]: I believe that such change could cause a choice conflict because busybox and bash would provide /bin/sh.

**Review by alexandrevicenzi — DISMISSED** [review]

File: src/bci_build/package/rmt.py (hunk 1 above)
Comment (dcermak) [reply src/bci_build/package/rmt.py:34]: That's one of the reasons why I added it

File: src/bci_build/package/rmt-server/entrypoint.sh [hunk 2]
```diff
@@ -23,30 +23,46 @@ MYSQL_USER="${MYSQL_USER:-rmt}"
...
+log_level:
+  rails: debug
+EOF
 
 if [ $# -eq 0 ]; then
 	set -- /usr/share/rmt/bin/rails server -e production
 fi
 
-if [ "$1" == "/usr/share/rmt/bin/rails" -a "$2" == "server" ]; then
+if [ "$1" == "/usr/share/rmt/bin/rails" ] && [ "$2" == "server" ]; then
```
Comment (dirkmueller) [inline src/bci_build/package/rmt-server/entrypoint.sh:51]:
```suggestion
if [ "$1" = "/usr/share/rmt/bin/rails" ] && [ "$2" = "server" ]; then
```

File: src/bci_build/package/rmt-server/entrypoint.sh [hunk 3]
```diff
@@ -23,30 +23,46 @@ MYSQL_USER="${MYSQL_USER:-rmt}"
...
   echo "Create/migrate SUSE RMT database"
   pushd /usr/share/rmt > /dev/null
 	/usr/share/rmt/bin/rails db:create db:migrate RAILS_ENV=production
   popd > /dev/null
-  if [ ${SCC_SYNC} == "true" ]; then
+  if [ "${SCC_SYNC}" == "true" ]; then
```
Comment (dirkmueller) [inline src/bci_build/package/rmt-server/entrypoint.sh:56]:
```suggestion
  if [ "${SCC_SYNC}" = "true" ]; then
```

**Review by dirkmueller — APPROVED** [review]

**Review by alexandrevicenzi — APPROVED** [review]

**Review by ngetahun — APPROVED** [review]
LGTM!

**Pull request merged**